src/main/java/mate.academy/service/ProductService.java

```diff
@@ -1,8 +1,10 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
we put Component on the implementation, not interface. That marks the class which instance you want to create